cypress/e2e/eholdings/eholdings-titles.cy.js

```diff
@@ -47,9 +50,11 @@ describe('eHoldings titles management', () => {
     });
   });
 
-  // TODO: https://issues.folio.org/browse/UIEH-1256
+  //TODO: https://issues.folio.org/browse/UIEH-1256
```

Comment:
Expected exception block, space or tab after '//' in comment

Reply:
fixed